package.json

```diff
@@ -3805,7 +3805,6 @@
             {
                 "command": "aws.amazonq.showTransformationPlanInHub",
                 "title": "Show Transformation Plan",
-                "icon": "$(book)",
```

Comment:
This button appears in a dropdown menu where the icons are not visible, so removing it.